modules/services/amberol.nix

```
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.services.amberol;
+
+in {
+  meta.maintainers = with lib.maintainers; [ surfaceflinger ];
+
+  options.services.amberol = {
+    enable = lib.mkEnableOption ''
+      Amberol music player as a daemon
+      Note, it is necessary to add
+      ```nix
+      programs.dconf.enable = true;
+      ```
+      to your system configuration for the daemon to work correctly'';
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.amberol;
+      defaultText = lib.literalExpression "pkgs.amberol";
+      description = "The `amberol` package to use.";
+    };
+
+    settings = {
+      recoloring = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        description = "UI recoloring using the album art";
+      };
+
+      replaygain = lib.mkOption {
+        type = lib.types.enum [ "album" "track" "off" ];
+        default = "track";
+        description = "ReplayGain mode";
+      };
+    };
+  };
```

Only use `settings` for RFC-42 style settings. Can simply do something like

```suggestion
    enableRecoloring = lib.mkOption {
      type = lib.types.bool;
      default = true;
      description = "UI recoloring using the album art";
    };

    replaygain = lib.mkOption {
      type = lib.types.enum [ "album" "track" "off" ];
      default = "track";
      description = "ReplayGain mode";
    };
```